src/flows/components/FlowCards.tsx

```diff
@@ -27,6 +34,18 @@ const NoMatches = () => {
 }
 
 const FlowCards: FC<Props> = ({flows, search}) => {
+  const [pinnedItems, setPinnedItems] = useState([])
+
+  useEffect(() => {
+    if (isFlagEnabled('pinnedItems') && CLOUD) {
+      getPinnedItems()
+        .then(res => setPinnedItems(res))
```

Comment:
what if we just set this as a set and we could do some o(1) look up like `set.has`. Would look something like:

`setPinnedItems(new Set(...res))` then later on just checking to see if `pinnedItems.has` or something